tests/components/homekit/test_accessories.py

```diff
@@ -157,6 +159,31 @@ async def test_home_accessory(hass, hk_driver):
     assert serv.get_characteristic(CHAR_MODEL).value == "Test Model"
 
 
+async def test_accessory_with_missing_basic_service_info(hass, hk_driver):
+    """Test HomeAccessory class."""
+    entity_id = "sensor.accessory"
+    hass.states.async_set(entity_id, "on")
+    acc = HomeAccessory(
+        hass,
+        hk_driver,
+        "Home Accessory",
+        entity_id,
+        3,
+        {
+            ATTR_MODEL: None,
+            ATTR_MANUFACTURER: None,
```

Comment:
Looks like these constants changed in #53703

Reply:
Yeah, so I have been trying to update those as I went along. Got indeed merge conflicts. Guess I missed some, meh.